### ISSUE TITLE
Mutation adds a UI view that breaks accessibility on iOS (hides all children)

### ISSUE THREAD

**Issue by Meira** [apollographql/react-apollo#2649]
**Intended outcome:**
Child items such as buttons should be accessible via VoiceOver.

**Actual outcome:**
iOS-only!
The mutation view is the only touchable thing on the screen; it is impossible to interact with anything on the screen while VoiceOver is active.

XCUI structure (selected element should have fields and buttons in it):
![screen shot 2018-12-10 at 17 35 58 copy](https://user-images.githubusercontent.com/1642618/49750334-7f925c80-fca2-11e8-88ce-a589c0d254d8.png)

**How to reproduce the issue:**
Use a mutation with a text field or button in it. Set the button's accessibility label and accessible=true.
Build an .app and install it on a device.
Run VoiceOver and try to select and tap the button.

**Version**

- apollo-client@2.2.3
- react-apollo@2.4.2
